lib/server.coffee

```diff
@@ -111,7 +113,6 @@ exports.request = (options = {}, outerCallback, onProgress) ->
 				progress(request(options))
 					.on('progress', onProgress)
 					.on('error', outerCallback)
-					.on('end', onProgress)
```

Comment:
After some experimentation, `end` is not even being called by `node-request-progress`. Also by checking their example, they use `close` to determine the final callback, so I think we're good like it is (I wasn't able to reproduce a case where `outerCallback` was called twice neither).